src/getValue.native.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+const getValue = (
+  event: SyntheticInputEvent<*>,
+  currentValue: any,
+  valueProp: any,
+) => {
+  if (event.nativeEvent) {
```

Comment:
I've extracted this bit from the previous version of `getValue`, but it seems wrong to me, I have 0 experience with React Native though, so cant validate it :/

maybe this should be just
```js
const getValue = (event, currentValue, valueProp) => event.nativeEvent.text
```
?

Reply:
I also have zero RN experience. Maybe we should find someone who does?

Reply:
cc @MichelDiz @rafaelcorreiapoli @geowarin @erickjth @bogdansoare @ibratoev @cosmith @maciejmyslinski @ghoshabhi

I think you guys were using (at least at some point in time) this library in RN. Could you take a look at this PR?

Reply:
I've never used react-final-form with react native. I asked my friends who code in react native for help, though.

Reply:
Cool! Sorry for pinging you then, I've looked through RN-related topics and pinged everyone who was involved.